cmd/catalog.go

```diff
@@ -34,7 +34,7 @@ var catalogListCmd = &cobra.Command{
 
 var catalogListComponentCmd = &cobra.Command{
 	Use:   "components",
-	Short: "List all available component types.",
+	Short: "List all components available.",
```

Comment:
I seem to get confused between component and component type
My current understanding is that a component is an application built using a builder imagestream of type same as component type.. which makes me arrive at 
component === app
component type === builder image stream of respective type..
It is possible that I am missing something
Please correct me if I am wrong...

Reply:
* App is your application which consist of multiple components
* Components is one of the part of your app
* component type is type of component, for example, java or python